libraries/disp3D/engine/view/view3D.cpp

```diff
@@ -494,7 +494,7 @@ void View3D::initMultiView()
 
     updateMultiViewAspectRatio();
 
-    auto noDraw = new Qt3DRender::QNoDraw(clearBuffers);
```

Comment:
@gabrielbmotta this one I don't really understand why it was created.

Reply:
This may have been leftover from the QML guide I found and attempting to replicate in c++. I don't know if this is still necessary, might have to test it with that line commented out and see if the rendering is working correctly